### PR TITLE
Pass arguments to SIF decoder

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -46,3 +46,12 @@ for p in problems
   end
   cutest_finalize(nlp)
 end
+
+# test arguments passed to decoder
+model = CUTEstModel("DIXMAANJ", "-param", "M=5")
```
Comment (abelsiqueira) [inline test/runtests.jl]: These should be `nlp` or L53 and L57 should be `model`.

Comment (dpo) [inline test/runtests.jl]: Gee, thanks! :-}

**Comment by abelsiqueira** [comment]
Thanks

**Comment by pkofod** [comment]
Awesome!

**Comment by dpo** [comment]
That gives access to A LOT more problems!

**Pull request merged**